server/src/main/webapp/WEB-INF/rails/webpack/views/components/switch/index.tsx

```diff
@@ -59,14 +60,15 @@ export class SwitchBtn extends MithrilViewComponent<Attrs> {
                    vnode.attrs.field(target.checked);
                  }}
                  class={styles.switchInput}
-                 data-test-id="switch-checkbox"/>
+                 data-test-id={dataTestId}/>
```

Comment:
I like this -- tests are more resilient and it communicates intent. 👍

Reply:
Thanks!